platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functionnal/SOCNotificationIntranetMentionTestIT.java

```diff
@@ -0,0 +1,281 @@
+        navigationToolbar.goToAddUser();
+        userAddManagement.addUser(username1, "123456", email1, username1, username1);
+        userAddManagement.addUser(username2, "123456", email2, username2, username2);
+        manageLogInOut.signIn(username1, "123456");
+        homePagePlatform.goToHomePage();
+        String activity1 = "activitya" + getRandomNumber();
+        activityStream.mentionUserActivity(username2, activity1);
+        manageLogInOut.signIn(username2, password);
+        navigationToolbar.goToIntranetNotification();
+        arrayUser.add(username1);
+        String status = "has mentioned you";
+        intranetNotification.checkAvatarInStatus(username1 + " " + username1, true);
+        intranetNotification.checkStatus(status, username1);
+        intranetNotification.checkActivityTitleInStatus(activity1, true);
+        intranetNotification.checkUsers(arrayUser, true);
+
+        info("Test 4: Click the Mention notifications (in activity message)");
+        intranetNotification.checkAvatarInStatus(arrayUser, true);
+        intranetNotification.checkStatus(status, username1);
+        intranetNotification.checkActivityTitleInStatus(activity1, true);
+        refresh();
+        $(byXpath(ELEMENT_ACTIVITY_MENTION_USER.replace("${content}", activity1).replace("${user}", username2))).waitUntil(Condition.visible, Configuration.timeout);
+        manageLogInOut.signIn(DATA_USER1, "gtngtn");
+        navigationToolbar.goToManageCommunity();
+        addUsers.deleteUser(username1);
+        addUsers.deleteUser(username2);
+    }
+
+    /*Step Number: 1
+            *Step Name:
+            *Step Description:
+               - Login with User B
+               - Click the notification icons in the top navigation
+               - Check the notification list
+            *Input Data:
+
+            *Expected Outcome:
+               - The Mention notification is displayed in the list*/
+    /*Step number: 2
+		 *Step Name:
+		 *Step Description:
+			- Check the notification message
+		 *Input Data:
+
+		 *Expected Outcome:
+			The notification message is : $AVATAR$USER has mentioned you $ACTIVITY$DATEWhere :
+			- $AVATAR is the thumbnail of User A
+			- $USER is User A
+			- $ACTIVITY is the name of the wiki page
+			- $DATE is the date of the notification*/
+    /*Step Number: 1
+		 *Step Name:
+		 *Step Description:
+			- Login with User B
+			- Click the notification icons in the top navigation
+			- Check the notification list
+		 *Input Data:
+
+		 *Expected Outcome:
+			- The Mention notification is displayed in the list*/
+
+    /*Step number: 2
+     *Step Name:
+     *Step Description:
+        - Click the notification
+     *Input Data:
+
+     *Expected Outcome:
+        - The user is redirected to the activity viewer with all comment expanded.
+        - The comment where the mention has been done is highlighted*/
+    @Test
+    public void test02_CheckMentionNotificationsInComment() {
```

Comment:
This function contains 2 tests, you must split them

Reply:
done